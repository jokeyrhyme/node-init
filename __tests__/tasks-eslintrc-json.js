/* @flow */
'use strict';

const {
  lib: { eslintrcUpdater },
} = require('../lib/tasks/eslintrc.json.js');

test('eslintrcUpdater() fresh with react', () => {
  const rc = {};
  eslintrcUpdater(rc, { hasReact: true });
  expect(rc).toMatchSnapshot();
});

test('eslintrcUpdater() fresh without react', () => {
  const rc = {};
  eslintrcUpdater(rc, { hasReact: false });
  expect(rc).toMatchSnapshot();
});
