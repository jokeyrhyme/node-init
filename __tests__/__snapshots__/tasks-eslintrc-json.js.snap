// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslintrcUpdater() fresh with react 1`] = `
Object {
  "extends": Array [
    "eslint:recommended",
    "plugin:node/recommended",
    "plugin:react/recommended",
  ],
  "plugins": Array [
    "import",
    "node",
    "promise",
    "react",
  ],
}
`;

exports[`eslintrcUpdater() fresh without react 1`] = `
Object {
  "extends": Array [
    "eslint:recommended",
    "plugin:node/recommended",
  ],
  "plugins": Array [
    "import",
    "node",
    "promise",
  ],
}
`;
